src/mlpack/core/data/impute_strategies/custom_strategy.hpp

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file custom_strategy.hpp
+ * @author Keon Kim
+ *
+ * Definition and Implementation of the empty CustomStrategy class.
+ */
+#ifndef MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_STRATEGY_HPP
+#define MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_STRATEGY_HPP
+
+#include <mlpack/core.hpp>
+
+
+using namespace std;
+
+namespace mlpack {
+namespace data {
+
+class CustomStrategy
+{
+  // empty class
+};
+
+} // namespace data
+} // namespace mlpack
+
```

Comment:
What is this class for?Do nothing?In this case, maybe name it as DoNothingStrategy is better?